utils/markdown-link-attributer.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-param-reassign */
+
+// Ref. https://github.com/crookedneighbor/markdown-it-link-attributes/blob/7e83cca/index.js
```

Comment:
Yes. This snippet is inspired by `markdown-it-link-attributes`. 

Since we have some specific rules for the anchor tag rendering. I try to make it general and put it here. For the specific rules, I put it on `markdown-loader.js` so that this snippet can be reused later.

I can see the `markdown-loader.js` is getting complex. Would it be better to extract those rules to another file?